exercises/acronym/package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "exercism-javascript",
-  "version": "0.0.0",
```

Comment:
These changes are because the script deletes `version` from copied files [here](https://github.com/exercism/javascript/blob/032fdbd4b7868757a8f1123861e144b433b5d2b7/scripts/helpers.js#L49).